packages/builder/src/components/common/bindings/BindingPanel.svelte

```diff
@@ -77,20 +78,27 @@
     Evaluation: "play",
   }
 
-  $: editorModeOptions = getModeOptions(allowHBS, allowJS)
+  $: editorModeOptions = getModeOptions(allowHBS, allowJS, allowHTML)
   $: sidePanelOptions = getSidePanelOptions(bindings, context)
   $: enrichedBindings = enrichBindings(bindings, context, snippets)
   $: usingJS = mode === BindingMode.JavaScript
   $: editorMode =
-    mode === BindingMode.JavaScript ? EditorModes.JS : EditorModes.Handlebars
+    mode === BindingMode.JavaScript
```

Comment:
Can we update this so it's not a nested ternary? I find them quite hard to reason about.

Reply:
done